.circleci/config.yml

```diff
@@ -38,14 +48,50 @@ workflows:
 
 jobs:
   build:
+    docker:
+      - image: golang:1.12
+    working_directory: /go/src/github.com/googleapis/gapic-showcase
+    steps:
+      - checkout
+      - run:
+          name: Get proto dependencies
+          command: git submodule init && git submodule update
```

Comment:
curious, will we remove the submodule after all the necessary annotations are merged into master? Or will this persist?

Reply:
Merged into master of googleapis/googleapis? I think keeping the submodule makes sense since it shows where the dependencies of the schemas come from. I'll probably move the submodule into schema though to make that even more apparent.

Reply:
api-common-protos master, but yeah. 

Yeah moving it to the schema directory seems like a good idea imo.

Reply:
Moving to schema in #112.